### PR TITLE
Ubuntu 20.04

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

File: build-scripts/ubuntu-1604/prepare-package.sh [hunk 1]
```diff
@@ -35,8 +35,6 @@ sed -i -r "s~sovrin-dev~sovrin~" "$repo/setup.py"
 
 echo -e "\n\nPrepare debian package versions"
 sed -i -r "s~indy-node==([0-9\.]+[0-9])(\.)?([a-z]+)~indy-node==\1\~\3~" "$repo/setup.py"
-sed -i -r "s~sovtoken==([0-9\.]+[0-9])(\.)?([a-z]+)~sovtoken==\1\~\3~" "$repo/setup.py"
```
Comment (devinleighsmith) [inline build-scripts/ubuntu-1604/prepare-package.sh:38]: can't hold sovtoken, sovtokenfees, libsovtoken until the 20.04 versions are available.

**Comment by devinleighsmith** [comment]
This PR requires https://github.com/sovrin-foundation/sovrin/pull/294 to be merged first.

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+      INDY_PLENUM_VERSION: 1.12.2
+      INDY_NODE_VERSION: 1.12.2~rc1
+      SOVRIN_VERSION: ${{ needs.setup-workflow.outputs.PACKAGE_VERSION }}
+      SOVRIN_INSTALL: true
+
+    steps:
+      - name: Check out test automation
+        uses: actions/checkout@v2
+        with:
+          repository: devinleighsmith/indy-test-automation
```
Comment (devinleighsmith) [inline .github/workflows/build.yaml:99]: this will need to be fixed when the corresponding PR for this is merged.

**Comment by WadeBarnes** [comment]
(ci) test this please

File: .github/workflows/build.yaml (hunk 2 above)
Comment (WadeBarnes) [reply .github/workflows/build.yaml:99]: Looks like the PR was merged.

**Pull request closed without merging**